### PR TITLE
Added german text delegate

### PR DESCRIPTION
**Pull request by Letalus**

### CODE REVIEW

File: lib/src/delegates/assets_picker_text_delegate.dart [hunk 1]
```diff
@@ -194,3 +194,49 @@ class HebrewTextDelegate implements AssetsPickerTextDelegate {
   String durationIndicatorBuilder(Duration duration) =>
       AssetsPickerTextDelegate.defaultDurationIndicatorBuilder(duration);
 }
+
+
+
+/// [AssetsPickerTextDelegate] implements with German.
+/// 英文文字实现
```
Comment (AlexV525) [inline lib/src/delegates/assets_picker_text_delegate.dart:201]: You can use german to describe the delegate here.

File: lib/src/delegates/assets_picker_text_delegate.dart [hunk 2]
```diff
@@ -194,3 +194,49 @@ class HebrewTextDelegate implements AssetsPickerTextDelegate {
   String durationIndicatorBuilder(Duration duration) =>
       AssetsPickerTextDelegate.defaultDurationIndicatorBuilder(duration);
 }
+
+
```
Comment (AlexV525) [inline lib/src/delegates/assets_picker_text_delegate.dart:198]:
```suggestion
```

**Comment by Letalus** [comment]
Alright thanks for the suggestions. I hope those new translation are what you were looking for. Greetings Chris

File: lib/src/delegates/assets_picker_text_delegate.dart [hunk 3]
```diff
@@ -194,3 +194,49 @@ class HebrewTextDelegate implements AssetsPickerTextDelegate {
...
+
+/// [AssetsPickerTextDelegate] implementiert mit der deutschen Übersetzung.
+/// Deutsche Textimplementierung
```
Comment (AlexV525) [inline lib/src/delegates/assets_picker_text_delegate.dart:201]:
```suggestion
/// Deutsche Textimplementierung.
```

**Review by AlexV525 — APPROVED** [review]
LGTM. Take another look see if there are anything wrong.

**Comment by AlexV525** [comment]
@all-contributors Add @Letalus for translation.

**Pull request merged**